supplementary_style_guide/style_guidelines/links.adoc

```diff
@@ -2,8 +2,70 @@
 [[links]]
 = Links
 
+The following guidelines improve findability and search engine optimization.
+
+.Technical best practices
```

Comment:
This is a really good point @jlprevatt  For example, I don't know what an iframe link is. These technical points came from not-docs-specific, general guidance on links so I don't have examples.
I'm wondering if it's better to replace these with a 'do' entry that says something like -use the/<x> standard link format- but then not everyone uses asciidoc.

Reply:
I'm not a link expert, but I did a little bit of googling:
It looks like an iframe link would use/be enclosed in an <iframe></iframe> structure, a flash link would use an <object><embed></embed></object> structure, and a javascript link would use a <script></script> structure. 
Someone more savvy than I am, like JP Sherman, might be able to confirm if those generalizations are correct. If we can get examples that are that simple, then maybe those instructions won't be too confusing.

Reply:
Paging @jpsherman Please could you help by providing some good and simple examples of these link formats so that writers can recognize and avoid them? (or point me in the right direction would work too)

* Do not use JavaScript links. 
* Do not use Flash links.
* Do not use links in Frames or iframes.

Thanks!

Reply:
Hi @jlprevatt  I consulted JP on this and added examples. Does it work for you?
He did say that he doubts we can even insert non-html links into our docs content but this will make it extra "doubleplus" sure and clear in case it's possible somewhere.

Reply:
Not sure I saw this as a notification, or I have a lot of notifications coming in and I missed it. I think the added examples help.